velox/vector/BaseVector.h

```diff
@@ -756,6 +756,26 @@ class BaseVector {
     memoDisabled_ = true;
   }
 
+  // Set of options that validate() accepts.
+  struct ValidateOptions {
+    // If set to true then an unloaded lazy vector is loaded and validate is
+    // called on the loaded vector. NOTE: loading a vector is non-trivial and
+    // can modify how it's handled by downstream code-paths.
+    bool loadLazy;
+
+    // Any optional checks you want to execute on the vector.
+    std::function<void(const BaseVector&)> callback;
+
+    ValidateOptions() {
```

Comment:
Just set a default argument to the field is enough, no need for a explicit constructor

Reply:
As discussed offline, I have moved it outside of the BaseVector class due to compiler issues with default initializing the member variables of this struct.

error was:
```
Default member initializer for 'loadLazy' needed within definition of enclosing class 'BaseVector' outside of member functions
```